testdata/dnn/download_models.py

```diff
@@ -178,6 +178,11 @@ def get_confirm_token(response):  # in case of large files
 
 
 models = [
+    Model(
+        name='RAFT',
+        url='https://drive.google.com/u/0/uc?id=1yWIbF4birkpZMhNNhqzy75d4EvJW-UXE&export=download',
```

Comment:
Please share a source of that link in comment (i.e. origin repository)

Reply:
The model is sourced from here https://github.com/Aser-Abdelfatah/opencv_zoo/tree/gsoc_RAFT_Model_ONNX/models/optical_flow_estimation_raft

Reply:
I mean it there an officially distributed model? There is a little chance that custom file will be available for a long time.

Reply:
I think it make sense to highlight all that details at least as comments before the Model block.